src/components/Tooltip/Tooltip.tsx

```diff
@@ -7,7 +7,6 @@ import {useToggle} from '../../utilities/use-toggle';
 import {Key} from '../../types';
 
 import {TooltipOverlay, TooltipOverlayProps} from './components';
-import styles from './Tooltip.scss';
```

Comment:
Nitpick: If the styles in Tooltip.scss are now only used in the Tooltip/components/TooltipOverlay should we move this css file in to that subcomponent?